cloud/home/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Home | Terramate Cloud
+description: Learn about the Home page of the Terramate Cloud dashboard
+---
+
+# Home Page
+
+The Home page provides a summary of your engagements in Terramate Cloud, including:
+
+- **Alerts**: Open alerts assigned to you.
+- **Pull Requests**: Open pull requests authored by you.
+- **Deployments**: Deployments in which you collaborated.
+
+![Home Page](../assets/home_tmc.png "Home Page")
+
+
+## Alerts
+
+The Alerts list displays details of alerts assigned to you, following the same layout as the [Alerts dashboard](../alerts/index.md).
+
+
+## Pull Requests
+
+The Pull Requests list shows [pull requests](../previews/index.md) you authored in repositories associated with your Terramate Cloud organization.
+
+
+## Deployments
+
+The Deployments list presents deployments where you contributed by pushing commits. The list shows:
+
+- **Status**: The current state of each deployment.
+- **Triggering Pull Request and Metadata**: Displays the pull request that initiated the deployment, including its title, the author's name, and the branch where it was merged.
+- **Associated Stack**: The stack linked to the deployment.
+- **Associated Repository**: The repository containing the deployment's code.
+- **Finished At**: A timestamp indicating how long ago the deployment completed.
+- **Duration**: The total runtime of the deployment.
```

Comment:
controversial but I think "run time" is more accurate here.
FYI: https://jamesrwilcox.com/runtime.html

but I searched our docs and found that we are using "runtime" and "run time" very inconsistently.
For example:
[here](https://github.com/terramate-io/terramate-docs/blob/8f2d69c07ac5fcf7db346ca0cfa8c8cae6550b22/cli/stacks/nesting.md?plain=1#L61) and [here](https://github.com/terramate-io/terramate-docs/blob/8f2d69c07ac5fcf7db346ca0cfa8c8cae6550b22/cli/stacks/index.md?plain=1#L40)

For me it's odd everytime I read "runtime" to mean the "execution time" (how much time it ran), because runtime has a very niche meaning in compsci.